src/core/data/infractions.hpp

```diff
@@ -18,6 +18,7 @@ namespace big
 		ATTACKING_WITH_GODMODE,
 		ATTACKING_WITH_INVISIBILITY,
 		ATTACKING_WHEN_HIDDEN_FROM_PLAYER_LIST,
+		CHAT_SPAM,
```

Comment:
Don't ever change the order of this enum, it will break the already saved database. Always append.